src/components/alerts/alerts.jsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box/box.jsx';
+import Button from '../button/button.jsx';
+
+import styles from './alert.css';
+
+const Alerts = ({
+    className,
+    message,
+    onCloseAlert
+}) => (
+    <Box
+        bounds="parent"
+        className={classNames(styles.alertsContainer, className)}
```

Comment:
I don't think you need `styles.alertsContainer` anymore, it's not defined in `alert.css`.